apps/src/music/player/MusicLibrary.ts

```diff
@@ -401,18 +407,39 @@ export interface SampleSequence {
 
 export interface SoundData {
   name: string;
-  path?: string;
   src: string;
-  length: number;
-  type: SoundType;
-  note?: number;
+  path?: string;
   restricted?: boolean;
   sequence?: SampleSequence;
+  type?: SoundType;
   bpm?: number;
   key?: Key;
+  note?: number;
+  length?: number;
+}
+
+export interface SoundDataPack extends Omit<SoundData, 'sequence' | 'note'> {
```

Comment:
Now that we have these broken up, it might be cleaner to remove the `Omit` and just define them separately? Something like
```
// base sound data
interface SoundData {
  name: string;
  src: string;
  path?: string;
  skipLocalization?: boolean;
}

interface SoundDataPack extends SoundData {
  length: number;
  type: SoundType;
  ... restricted, bpm, key, etc
}

interface SoundDataInstrument extends SoundData {
  note: number;
}

interface SoundDataKit extends SoundData {
  length: number;
  // we don't have notes right now, but it would be great to add notes here too. Currently we just assign note based on index
}